src/screens/Booking/components/Baggage/Baggage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import {Divider} from 'react-native-paper';
+import {t} from '@translations';
+import styles from './Baggage.styles';
+import BaggageItem from '../BaggageItem/BaggageItem';
+
+interface BaggageProps {
+  cabin: Boolean;
+  checked: Boolean;
+  setCabinValue: () => void;
+  setCheckedValue: () => void;
+}
+
+const Baggage = ({
+  cabin,
+  checked,
+  setCabinValue,
+  setCheckedValue,
+}: BaggageProps) => (
+  <View style={styles.mainContainer}>
+    <Divider />
+    <View style={styles.container}>
+      <Text style={styles.sectionTitle}>{t.baggage.title}</Text>
+      <BaggageItem
```

Comment:
there is no implemtation that reflects this requirement "Select “Cabin Bag 40 x 18 x 25 cm, 7kg Free”
Select “Checked Bag 55 x 20 x 40 cm, 10kg Free”